spec/board_spec.rb

```diff
@@ -0,0 +1,83 @@
+# rubocop: disable Lint/ParenthesesAsGroupedExpression
+
+# rubocop: disable Lint/UselessAssignment
+
+require_relative '../lib/board'
+require_relative '../lib/player'
+
+describe Board do
+  context 'initialize' do
+    let(:board1) { Board.new }
+    it 'Creat the board array' do
+      expect(board1.board).to eq((1..9).to_a)
+    end
+  end
+
+  context 'display_board' do
+    let (:board1) { Board.new }
+    it 'Print the board' do
+      expect(board1.display_board).to eq("    1 | 2 | 3\n    ..|...|..\n    4 | 5 | 6\n    ..|...|..\n    7 | 8 | 9")
+    end
+  end
+
+  context 'full' do
+    let (:board1) { Board.new }
+    it 'Check if the board is full' do
+      expect(board1.full).to eq(false)
+    end
+  end
+
+  context 'win?' do
+    let (:board1) { Board.new }
+    it 'Checks if there is a winner by checking the patterns' do
+      expect(board1.win?).to be false
+    end
```

Comment:
## Negative scenario

- Specs cover positive and negative scenarios  (screenshot  from  Pathwright)
Example: there is a spec for correct usage of the method [e.g. it "creates object if all required attributes are passed"]
and also there is spec for the wrong usage of a method  [e.g. it "does not create the object if required attributes are missing"].

  - [ ] Thus, please add negative scenarios

Example:

```ruby
#positive scenario
 it 'Checks if there is a winner by checking the patterns' do
#negative scenario
it 'Checks if there is no a winner by checking the patterns' do
```